lokerpc/schema.go

```diff
@@ -71,6 +71,7 @@ func TypeSchema(t reflect.Type, defs map[string]jtd.Schema) *jtd.Schema {
 	case reflect.Map:
 		vals := TypeSchema(t.Elem(), defs)
 		schema.Values = vals
+		schema.Nullable = true
```

Comment:
it is a bit crap that clients will need to check non-null for this

not sure what the solution is, I guess the package needs to actually ensure that for the type to be that way